Dockerfile

```diff
@@ -2,15 +2,35 @@ FROM python:3.6-slim-jessie
 ENV PYTHONUNBUFFERED 1
 RUN mkdir /code && \
     apt-get update && \
-    apt-get install -y --no-install-recommends dos2unix gcc libc6-dev libc-dev libssl-dev make automake libtool autoconf pkg-config libffi-dev && \
+    apt-get install -y --no-install-recommends wget git dos2unix gcc libc6-dev libc-dev libssl-dev make automake libtool autoconf pkg-config libffi-dev && \
     pip3 install dumb-init && \
     rm -rf /var/lib/apt/lists/*
+
+RUN git clone --recursive https://github.com/maxmind/libmaxminddb.git && \
+    cd libmaxminddb && \
+    ./bootstrap && \
+    ./configure && \
+    make && \
+    make check && \
+    make install && \
+    echo /usr/local/lib  >> /etc/ld.so.conf.d/local.conf && \
+    ldconfig
+
+# GeoIP2 Data Files
+RUN mkdir -p /usr/share/GeoIP/ && \
```

Comment:
hopefully these steps dont take too long and/or they're not too brittle!

Reply:
I wouldn't say they're brittle, but it does add a little extra time to the build... but they are in their own cache layer, so the user should only need to pull these files once per build environment (until they clear docker cache/system images).